### PR TITLE
Change PR body when file size reduced by zero percent

### LINKED ISSUE #120: PR Body: Handle case where the total rounds down to 0%
image

Let's rewrite to "Your file size has been reduced! " in the case of 0

### PR DESCRIPTION
**Pull request by RohanNagar**
Closes #120.

### CODE REVIEW

**Comment by dlemstra** [comment]
Wonder if it wouldn't be nicer to print 0.3% when the rounded value is zero and otherwise print a rounded percentage e.g 1%?

File: OpenPrFunction/PullRequestBody.cs [hunk 1]
```diff
@@ -56,7 +56,16 @@ public static string Generate(string commitBody)
                 var sb = new StringBuilder();
                 sb.AppendLine("## Beep boop. Your images are optimized!");
                 sb.AppendLine();
-                sb.AppendLine($"Your image file size has been reduced by **{imageStats[0].Percent:N0}%** 🎉");
+
+                if (Math.Round(imageStats[0].Percent) == 0)
```
Comment (dabutvin) [inline OpenPrFunction/PullRequestBody.cs]: :+1:

File: Test/PullRequestBodyTests.cs [hunk 2]
```diff
@@ -51,6 +51,32 @@ public void GivenMultiImageCommitMessage_ShouldFormatMarkdownTable()
             Assert.AreEqual(expectedMarkdown, result);
         }
 
+        [TestMethod]
+        public void GivenZeroPercentReductionCommitMessage_ShouldOmitPercentage()
```
Comment (dabutvin) [inline Test/PullRequestBodyTests.cs]: :+1:

**Comment by dabutvin** [comment]
@RohanNagar this looks great! 

@dlemstra I agree that we could add that kind of logic. Hadn't thought about it when I wrote up the issue. I'll file a new request to make the change. This PR gives us the hook we need to build on. We had some other issues logged a while back about converting from KB to MB in some cases too

**Pull request merged**